src/components/content/Start.tsx

```diff
@@ -35,28 +41,58 @@ class Landing extends React.PureComponent<WithTranslation, {}> {
           <MuiMarkdown
             markdown={this.getMarkdown(this.props.i18n.language)}
           ></MuiMarkdown>
-          <Box marginTop={4}>
-            <Grid container spacing={4}>
-              <InfoBox
-                title={this.props.t("initiative.box1.title")}
-                content={this.props.t("initiative.box1.content")}
-              />
-              <InfoBox
-                title={this.props.t("initiative.box2.title")}
-                content={this.props.t("initiative.box2.content")}
-              />
-              <InfoBox
-                title={this.props.t("initiative.box3.title")}
-                content={this.props.t("initiative.box3.content")}
-              />
+        </Box>
+
+        <Box paddingBottom={4} marginTop={4}>
+          <Grid container spacing={4}>
+            <Grid item xs={12} sm={6} md={4} xl={3}>
+              <StyledLink to={`/${this.props.i18n.language}/initiative`}>
```

Comment:
should we externalise the links into the json files?

Reply:
Unless we would introduce localized routes (e.g. `/database` in en and `/datenbank` in de) I would not handle it in the localization files